mitodl/mitxpro
# add heading field in who should enroll section

#### What are the relevant tickets?
Fixes #855 

#### What's this PR do?
Add `heading` field in who should enroll section, that can be customized.

#### How should this be manually tested?
Add who should enroll section in cms, you should see a default heading field which can be customized.
